RELEASE-NOTES.md

```diff
@@ -1,3 +1,84 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep. This issue does not affect multipart
+  uploads.
+- Fix wrong path rewrite in S3 API by unnecessary unquote
+  [riak_cs/#1040](https://github.com/basho/riak_cs/pull/1040). Due to
+  the wrong handling of URL encoding/decoding, object keys including
```

Comment:
Two phrases for the same thing: "URL decoding/encoding" and "unquote/quote".
It would be better to choose one in order not to confuse reader.

I guess URL encoding/decoding is more usual than (un)quote, isn't it?